perfil/mandate/models.py

```diff
@@ -47,3 +47,9 @@ class Meta:
         indexes = [
             models.Index(fields=['position']),
         ]
+
+
+class PartyFiliation(models.Model):
+    party = models.ForeignKey(Party, on_delete=models.CASCADE)
+    person = models.ForeignKey(Person, on_delete=models.CASCADE,
+                               related_name='filiations')
```

Comment:
Do we have data to keep the start and end date of the affiliation?

Reply:
Not now, I just gathered the information about the party. We can add a issue for this

Reply:
Great!